record-core/src/iosMain/kotlin/dev/theolm/record/RecordCore.ios.kt

```diff
@@ -49,16 +38,17 @@ internal actual object RecordCore {
     @OptIn(ExperimentalForeignApi::class)
     @Throws(RecordFailException::class)
     internal actual fun startRecording(config: RecordConfig) {
-        println(config.toString())
         checkPermission()
         configureAudioSession()
 
         output = config.getOutput()
 
         val settings = mapOf<Any?, Any>(
             AVFormatIDKey to config.outputFormat.toAVFormatID(),
-            AVSampleRateKey to SampleRate,
-            AVNumberOfChannelsKey to 1,
+            AVSampleRateKey to config.sampleRate,
+            AVNumberOfChannelsKey to 1, // you may want to put a condition here?
```

Comment:
Is this comment for me or for you?
What condition?

Reply:
if you want to use 2 channels for the audio file, if you want to stick to one channel no need for conditions